doc/sphinx/source/esmvaldiag/observations.inc

```diff
@@ -1,7 +1,267 @@
+that are necessary to write the filename correctly and to understand which
+variable is of interest here. Please make sure to provide the correct
+information for following key words: DIAG_SCRIPT, VAR, NAME, MIP, FREQ,
+CMOR_TABLE.
+
+- **Note:** the fields ``VAR``, ``NAME``, ``MIP`` and ``FREQ`` all ask for one
+  or more entries. If more than one entry is provided, make sure that the order
+  of the entries is the same for all four fields! (for example, that the first
+  entry in all four fields describe the variable ``xch4`` that you would like
+  to extract);
+- **Note:** some functions in the script are NCL-specific and are available
+  through the loading of the script interface.ncl_. There are similar
+  functions available for python scripts.
+  
+.. _interface.ncl: https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/interface.ncl  
+
+.. _utilities.ncl: https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/cmorizers/obs/utilities.ncl
+
+In the second part of the script each variable defined in ``VAR`` is separately
+extracted from the original data file and processed. Most parts of the code are
+commented, and therefore it should be easy to follow. ESMValTool provides a set 
+of predefined utilities.ncl_, which can be imported into your CMORizer 
+by
+
+.. code-block:: NCL
+
+   loadscript(getenv("esmvaltool_root") + "/esmvaltool/cmorizers/obs/utilities.ncl")
+
+This module contains different kinds of small fixes to the data attributes,
+coordinates, and metadata which are necessary for the data field to be
+CMOR-compliant.
+
+5. Run the cmorizing script
+===========================
+
+The cmorizing script for the given dataset can be run with:
+
+.. code-block:: console
+
+ cmorize_obs -c <config-user.yml> -o <dataset-name>
+
+
+.. note::
+
+   The output path given in the configuration file is the path where
+   your cmorized dataset will be stored. The ESMValTool will create a folder
+   with the correct tier information (see Section `2. Edit your configuration file`_) if that tier folder is not
+   already available, and then a folder named after the data set. In this
+   folder the cmorized data set will be stored as a netCDF file. 
+
+If your run was successful, one or more NetCDF files are produced in your
+output directory.
+
+
+6. Naming convention of the observational data files
+====================================================
+
+For the ESMValTool to be able to read the observations from the NetCDF file,
+the file name needs a very specific structure and order of information parts
+(very similar to the naming convention for observations in ESMValTool
+v1.0). The file name will be automatically correctly created if a cmorizing
+script has been used to create the netCDF file.
+
+The correct structure of an observational data set is defined in 
+``config-developer.yml``, and looks like the following:
```

Comment:
Maybe you can make this a link to the section describing the config-developer.yml file? It may be hard to find otherwise

Reply:
Done.